common/lib/capa/capa/responsetypes.py

```diff
@@ -813,12 +826,22 @@ class ChoiceResponse(LoncapaResponse):
     def setup_response(self):
         self.assign_choice_names()
 
-        correct_xml = self.xml.xpath('//*[@id=$id]//choice[@correct="true"]',
-                                     id=self.xml.get('id'))
+        correct_xml = self.xml.xpath(
+            '//*[@id=$id]//choice[@correct="true"]',
+            id=self.xml.get('id')
+        )
 
         self.correct_choices = set([choice.get(
             'name') for choice in correct_xml])
 
+        incorrect_xml = self.xml.xpath(
+            '//*[@id=$id]//choice[@correct="false"]',
+            id=self.xml.get('id')
+        )
+
+        self.incorrect_choices = set([choice.get(
+            'name') for choice in incorrect_xml])
```

Comment:
This was adapted from line 835 above. Happy to change if it would help.

I feel like my next project should be fixing all the pep8/pylint violations, hard-to-read code, and one-off bizarro functions that I found while adapting the existing code.